diff --git a/test/test_dwarfdumpsetup.sh b/test/test_dwarfdumpsetup.sh
index 354ea0fe..d4e99a61 100755
--- a/test/test_dwarfdumpsetup.sh
+++ b/test/test_dwarfdumpsetup.sh
@@ -55,7 +55,13 @@ fi
 # prints differently for different time zones.
 textlim=700
 cp "$top_srcdir/src/bin/dwarfdump/dwarfdump.conf" .
-dd=$top_blddir/src/bin/dwarfdump/dwarfdump
+# prefix PATH with build location of dwarfdump
+# if configure --disable-dwarfdump this will only work
+# tests will only work if dwarfdump is installed somewhere
+# else in the path already and working
+export PATH=$top_blddir/src/bin/dwarfdump:$PATH
+dd=dwarfdump
+echo -n "Testing dwarfdump @ "; which dwarfdump
 
 # Delete what follows 'last time 0x0'
 fixlasttime() {
